.eslintrc.json

```diff
@@ -84,10 +84,6 @@
         "key-spacing": "error",
         "keyword-spacing": "error",
         "line-comment-position": "error",
-        "linebreak-style": [
-            "error",
-            "windows"
-        ],
```

Comment:
Instead of taking it out please change it to "linebreak-style": "off"

Reply:
resolved